tests/installation/releasenotes_online.pm

```diff
@@ -0,0 +1,37 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Check origin of displayed releasenotes during installation
```

Comment:
As a suggestion 'release_notes_from_rpm' and 'release_notes_from_url' would reflect better what is about the test without opening the code than 'online' or 'offline'. In other words, as I will schedule those test modules in Online or Offline, as a tester I already have that info, so the name of the test should include the origin somehow.